### ISSUE TITLE
Synchronize Scroll

### ISSUE THREAD

**Issue by jbickelhaupt** [mjmlio/mjml-app#159]
Would it be easy to add a synchronize scroll option like in the Try-It-Live editor for web browsers?

**Comment by meriadec** [comment]
Yes, good idea, that's one of the thing that is missing compared to try-it-live.
